app/scripts/lib/account-tracker.js

```diff
@@ -42,26 +30,34 @@ import { previousValueComparator } from './util';
  * @typedef {object} AccountTracker
  * @property {object} store The stored object containing all accounts to track, as well as the current block's gas limit.
  * @property {object} store.accounts The accounts currently stored in this AccountTracker
+ * @property {object} store.accountsByChainId The accounts currently stored in this AccountTracker keyed by chain id
  * @property {string} store.currentBlockGasLimit A hex string indicating the gas limit of the current block
+ * @property {string} store.currentBlockGasLimitByChainId A hex string indicating the gas limit of the current block keyed by chain id
  * @property {object} _provider A provider needed to create the EthQuery instance used within this AccountTracker.
- * @property {EthQuery} _query An EthQuery instance used to access account information from the blockchain
  * @property {BlockTracker} _blockTracker A BlockTracker instance. Needed to ensure that accounts and their info updates
  * when a new block is created.
- * @property {object} _currentBlockNumber Reference to a property on the _blockTracker: the number (i.e. an id) of the the current block
+ * @property {object} _currentBlockNumberByChainId Reference to a property on the _blockTracker: the number (i.e. an id) of the the current block keyed by chain id
```

Comment:
Do we want to preserve a version that is not keyed by chainId like we did with `currentBlockGasLimit` and `accounts`

Reply:
Not necessary since this is internal state on the controller itself and not the store, so it's not exposed to the rest of the UI and is not breaking

Reply:
right yeah makes sense